db/models.py

```diff
@@ -1,8 +1,15 @@
+import settings
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import UniqueConstraint
 
 
 class Genre(models.Model):
-    name = models.CharField(max_length=255, unique=True)
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        models.Index(fields=["title"])
```

Comment:
why do you make an index for this field? It is not required by the task

Reply:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/108864360/198566369-aba359ec-1306-48c5-97cf-b7eb97de5b9f.png">